elevation_mapping/include/elevation_mapping/sensor_processors/PerfectSensorProcessor.hpp

```diff
@@ -56,11 +56,10 @@ class PerfectSensorProcessor : public SensorProcessorBase
    * @param[out] variances the elevation map height variances.
    * @return true if successful.
    */
-	virtual bool computeVariances(
-			const pcl::PointCloud<pcl::PointXYZRGB>::ConstPtr pointCloud,
-			const Eigen::Matrix<double, 6, 6>& robotPoseCovariance,
-			Eigen::VectorXf& variances);
+   virtual bool computeVariances(
+		   const pcl::PointCloud<pcl::PointXYZRGB>::ConstPtr pointCloud,
```

Comment:
According to [this](https://leggedrobotics.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions) you would indent with 4 whitespaces. 
Please also make sure to not have tabs in there.